### PR TITLE
Taiwanese to Traditional Chinese

### PR DESCRIPTION
**Pull request by darcy-shen**
The usage of the word `taiwanese`  is not suitable in China. We rename Taiwanese to Traditional Chinese.

If you are in Taiwan, and you do not love this pull request, I have no superpower to prevent you making your hands dirty.

The language for Traditional Chinese is `Chineset` when the interface language is English. Currently, I have no good idea to make it show like `Traditional Chinese`. When the interface language is British, it is correctly showed as `Traditional Chinese`.

### CODE REVIEW

File: TeXmacs/progs/convert/latex/texout.scm [hunk 1]
```diff
@@ -328,7 +328,7 @@
 	((not (string-index x #\!)) (output-verb "\\verb!" x "!"))
 	((not (string-index x #\9)) (output-verb "\\verb9" x "9"))
 	((not (string-index x #\X)) (output-verb "\\verbX" x "X"))
-	(else (output-verb "\\verb�" x "�"))))
+	(else (output-verb "\\verb�" x "�"))))
```
Comment (PikachuHy) [inline TeXmacs/progs/convert/latex/texout.scm:331]: what's this?

File: TeXmacs/packages/customize/language/taiwanese.ts [hunk 2]
```diff
@@ -4,10 +4,10 @@
```
Comment (PikachuHy) [inline TeXmacs/packages/customize/language/taiwanese.ts:4]: do we need to delete this file?
```
TeXmacs/packages/customize/language/taiwanese.ts
```

File: TeXmacs/progs/convert/latex/texout.scm (hunk 1 above)
Comment (darcy-shen) [reply TeXmacs/progs/convert/latex/texout.scm:331]: in some scheme files, there are encodings which will be converted by vscode. Be careful!

**Review by PikachuHy — APPROVED** [review]
LGTM

**Comment by Iey4iej3** [comment]
This should also be adopted in the upstream TeXmacs. Traditional Chinese characters are also adopted in Hong Kong and Macau.

**Comment by darcy-shen** [comment]
@Iey4iej3 It is not OK for me to apply the same changes for two reasons:

1. This pull request is ugly and incomplete now
2. It is not a good idea for me to abuse my commit priviledge in the TeXmacs svn repo. Bringing politics to a open source project is very annoying.

**Pull request merged**